pkg/plugins/golang/v4/scaffolds/internal/templates/test/e2e/test.go

```diff
@@ -44,106 +44,101 @@ var TestTemplate = `{{ .Boilerplate }}
 package e2e
 
 import (
+	"context"
 	"fmt"
+	"github.com/go-resty/resty/v2"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
Nit: The imports must to be sorted out accordingly here.

Reply:
Sure, on it.